backend/ticketvise/models/ticket.py

```diff
@@ -219,3 +277,29 @@ def save(self, force_insert=False, force_update=False, using=None, update_fields
         """
         if self.receiver.notification_ticket_status_change_app or self.receiver.notification_ticket_status_change_mail:
             super().save(force_insert, force_update, using, update_fields)
+
+
+class TicketEvent(models.Model):
+    objects = InheritanceManager()
+    ticket = models.ForeignKey(Ticket, on_delete=models.CASCADE)
+    initiator = models.ForeignKey("User", models.CASCADE, null=True)
+    date_created = models.DateTimeField(auto_now_add=True)
+
+
+class TicketStatusEvent(TicketEvent):
+    old_status = models.CharField(max_length=8, choices=Status.choices, null=True)
+    new_status = models.CharField(max_length=8, choices=Status.choices, null=False)
+
+
+class TicketAssigneeEvent(TicketEvent):
+    assignee = models.ForeignKey("User", on_delete=models.CASCADE)
+
+
+class TicketLabelEvent(TicketEvent):
+    label = models.ForeignKey("Label", on_delete=models.CASCADE)
+    is_added = models.BooleanField()
+
+
+class TicketTitleEvent(TicketEvent):
+    old_title = models.CharField(max_length=100)
+    new_title = models.CharField(max_length=100)
```

Comment:
wanneer zou een title van een ticket veranderen, volgens mij is dat nu niet aan de orde dat je het kan editen